src/StreamrClient.js

```diff
@@ -50,21 +50,23 @@ export default class StreamrClient extends EventEmitter {
             retryResendAfter: 5000,
             gapFillTimeout: 5000,
             maxPublishQueueSize: 10000,
+            publisherGroupKeys: {}, // {streamId: groupKey}
+            subscriberGroupKeys: {}, // {streamId: {publisherId: groupKey}}
         }
         this.subscribedStreams = {}
 
         Object.assign(this.options, options || {})
 
         const parts = this.options.url.split('?')
         if (parts.length === 1) { // there is no query string
-            this.options.url = `${this.options.url}?controlLayerVersion=1&messageLayerVersion=30`
+            this.options.url = `${this.options.url}?controlLayerVersion=1&messageLayerVersion=31`
```

Comment:
Btw do we have the latest version numbers in some static fields in the protocol impl? I guess these numbers could come automatically from the used protocol package.